blue-note/theme.json

```diff
@@ -223,6 +223,48 @@
 					}
 				}
 			},
+			"core/comment-content": {
+				"css": " & a{text-decoration:underline;} & a:hover{text-decoration:none;}"
+			},
+			"core/comment-template": {
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "none"
```

Comment:
Removes the underline from the default state of the links elsewhere in the comment area.
Adds the underline on :hover.

**Note:** I did not change the text color because the suggest color has too low color contrast against the background color.